registry/domain/models/service_group.py

```diff
@@ -28,7 +28,7 @@ def to_dict(self):
     def _get_endpoints(self, endpoint_dict):
         endpoints = []
         for endpoint, val in endpoint_dict.items():
-            endpoints.extend(endpoint)
+            endpoints.append(endpoint)
```

Comment:
can we add a test case for this ?

Reply:
sure, i see for some reason test cases are commented , will enable it.

Reply:
added testcases